.github/workflows/main.yml

```diff
@@ -12,29 +12,14 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - name: Set up OCaml
-      uses: avsm/setup-ocaml@v1.0.1
-      with:
-        ocaml-version: 4.08.1 # optional, default is 4.08.1
-
-    - name: Setup Python
-      uses: actions/setup-python@v2
-
-    - name: Set up Coq
-      run: |
-        opam repo add coq-released https://coq.inria.fr/opam/released
-        opam update -y
-        opam install -y coq.8.11.1
-
-    - name: Clone bug minimizer
-      run: |
-        git clone https://github.com/JasonGross/coq-tools.git
-        sudo cp coq-tools/*.py /usr/bin/
-
     - name: Run minimizer
-      run: ./run.sh
+      uses: coq-community/docker-coq-action@v1
+      with:
+        opam_file: 'dummy.opam'
+        custom_image: 'registry.gitlab.com/coq/coq:bionic_coq-V2020-08-28-V92'
```

Comment:
These Docker images are regularly cleaned up from the GitLab registry by Emilio. Committing to using one such image for normal runs of the bug minimizer (as in `minimize now`) is probably not a good idea. What about using the default Docker-Coq-Action image for normal runs (latest stable Coq with minimal OCaml version) and *adding* this line when the minimizer is invoked to minimize a CI failure?

Reply:
Like this, it would act as a placeholder for the bot:

``` suggestion
         #custom_image: 'registry.gitlab.com/coq/coq:CACHEKEY'
```

Reply:
Done, thanks for the suggestion